neurons/validator.py

```diff
@@ -167,6 +167,7 @@ def sample_random_uids(
             List[int]: A list of responding and free uids.
         """
         active_uids = self.ping_all_miners(exclude_uids=exclude_uids)
+        return active_uids
```

Comment:
Should this be here?

Reply:
Good catch I though I committed only the if statement